src/components/SingleArticle/SingleArticle.jsx

```diff
@@ -0,0 +1,79 @@
+import Header from "../Header";
+import { useParams } from "react-router-dom";
+import { viewSingleArticle, getComments } from "../../utils/api";
+import { useState, useEffect } from "react";
+import "./SingleArticle.css";
+
+export default function SingleArticle() {
+  const { article_id } = useParams();
```

Comment:
good work implementing this to extract the id from the url bar, this means as a user I can change the article id and get the correct one on the screen.